src/components/atoms/NotFound/NotFound.tsx

```diff
@@ -9,6 +9,8 @@ import SPARKLEVERSE_LANDSCAPE from "assets/images/Sparkle_logo_white_yellow.png"
 
 import "./NotFound.scss";
 
+const JUST_CURRENT_PAGE_HISTORY_LENGTH = 1;
```

Comment:
No change required. Just a thought for you.

I know we love constants. In this case, I don't think this constant actually provides any value to us. It's not something we're ever going to change unless browsers change their history behaviour in a massive way.

Reply:
IMHO Constants aren't just for "change it in one place, see it reflected everywhere", they're also about providing meaningful names to values like 1, 3.14, "www.wherever.com"... 

Example, seeing   `NOT_FOUND === [].findIndex()` vs `-1 === [].findIndex()`, we know the -1 result will never change the meaning.

Reply:
Agree with "providing meaning" in most cases. In these specific cases, I'd hope the engineer knows what that constant means in that situation without needing explanation.

Reply:
Agree with "providing meaning" in most cases. In these specific cases, I'd hope the engineer knows what that constant means in that situation without needing explanation.